daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/TestMain.scala

```diff
@@ -93,6 +93,7 @@ object TestMain extends StrictLogging {
           case None =>
             val (apiParameters, cleanup) = if (config.ledgerHost.isEmpty) {
               val sandboxConfig = SandboxConfig.default.copy(
+                port = 0, // Automatically choose a free port.
```

Comment:
I guess `sandbox.port` points to the resolved port instead of `0`? Otherwise I don’t understand how we can connect.

Reply:
Yes, exactly. The `SandboxServer` object exposes the selected port. So, fortunately no need for `portFile` or the like.

Reply:
Great, thanks for the confirmation :+1: